README.md

```diff
@@ -1229,6 +1229,26 @@ Per capita personal income is total personal income divided by total midyear pop
 
 U.S. Bureau of Economic Analysis
 
+### <a id="table42"></a>Table 42. Funded Ratio of Public Pension Plans
+
+Fiscal Year 2014
+
+#### Explanation of Data
+
+| Name | Description |
+| --- | --- |
+| `id`  | The ANSI code identifying the state in each record. |
+| `stateAbbr` | The ANSI code abbreviation for the state in each record. |
+| `stateName` | The full name of the state in each record. |
+| `incomePerCapita` | Funded ratio. |
```

Comment:
As a non-economist, I'm not sure what this means. Could you explain it a bit more?